Sources/Vitamin/Foundations/Shadows/README.md

```diff
@@ -0,0 +1,40 @@
+## Shadows
+[Reference](https://www.decathlon.design/726f8c765/p/5522fa-shadows/b/69ed0e)
+
+Vitamin provides you with the `VitaminShadow`enum, which has five possible cases :
+- `.shadow100`
+- `.shadow200`
+- `.shadow300`
+- `.shadow400`
+- `.none`
+
+To apply a shadow to any view, you can call the `dropShadow(shadowType: VitaminShadow)` method on this view.
+
+```swift
+import Vitamin
+
+let label = UILabel.init(
```

Comment:
Please don't use `.init` when not necessary.
```suggestion
let label = UILabel(
```

Reply:
👍